core/src/mindustry/core/Renderer.java

```diff
@@ -30,6 +30,8 @@
     public @Nullable Bloom bloom;
     public FrameBuffer effectBuffer = new FrameBuffer();
     public float laserOpacity = 1f;
+    /** minZoom = zooming out, maxZoom = zooming in */
+    public float minZoom = 1.5f, maxZoom = 1.5f * 6;
```

Comment:
hold up, why is this 1.5*6 instead of 6

Reply:
oops i thought it was Scl.scl(1.5f) * 6f